client/src/commands/wrapLinesWithComment.ts

```diff
@@ -0,0 +1,80 @@
+import { Range, commands, window } from "vscode";
+
+/**
+ * Registers a command to wrap the non-blank content of the selected line or multiple lines with SAS block comments (/* ... *\/).
+ * If the selection spans multiple lines, each line will be commented individually.
+ */
+export function registerAddCommentCommand() {
+  return commands.registerCommand("SAS.addBlockComment", async () => {
+    const editor = window.activeTextEditor;
```

Comment:
there's a `commands.registerTextEditorCommand` API that will provide active editor in the callback and you won't need to check if editor exists.